src/Mod/Path/PathScripts/PathDressupHoldingTags.py

```diff
@@ -177,13 +183,13 @@ def createSolidsAt(self, z, R):
     def filterIntersections(self, pts, face):
         if type(face.Surface) == Part.Cone or type(face.Surface) == Part.Cylinder or type(face.Surface) == Part.Toroid:
             PathLog.track("it's a cone/cylinder, checking z")
-            return list(filter(lambda pt: pt.z >= self.bottom() and pt.z <= self.top(), pts))
+            return list([pt for pt in pts if pt.z >= self.bottom() and pt.z <= self.top()])
         if type(face.Surface) == Part.Plane:
             PathLog.track("it's a plane, checking R")
             c = face.Edges[0].Curve
             if (type(c) == Part.Circle):
-                return list(filter(lambda pt: (pt - c.Center).Length <= c.Radius or PathGeom.isRoughly((pt - c.Center).Length, c.Radius), pts))
-        print("==== we got a %s" % face.Surface)
+                return list([pt for pt in pts if (pt - c.Center).Length <= c.Radius or PathGeom.isRoughly((pt - c.Center).Length, c.Radius)])
+        PathLog.error("==== we got a %s" % face.Surface)
```

Comment:
Just a question: Which syntax is preferred moving forward for Python 3.7 and beyond?
1.)        PathLog.error("==== we got a %s" % face.Surface)
2.)        PathLog.error("==== we got a {}".format(face.Surface))